File: react-native/src/config/api.config.ts
export const API_CONFIG = {
  OPENAI: {
    BASE_URL: 'https://api.openai.com/v1',
    CHAT_COMPLETIONS: 'https://api.openai.com/v1/chat/completions',
    MODEL: 'gpt-3.5-turbo',
    TEMPERATURE: 0.3,
    MAX_TOKENS: 1000,
    SEED: 6,
  },
  TIMEOUTS: {
    DEFAULT: 30000, 
    UPLOAD: 60000, 
  },
} as const;

File: ios/FlashcardAI/Config/APIConfig.swift
import Foundation

struct APIConfig {
    struct OpenAI {
        static let chatCompletionsURL = "https://api.openai.com/v1/chat/completions"
        static let model = "gpt-3.5-turbo"
        static let temperature: Double = 0.3
        static let maxTokens = 1000
        static let seed = 6
    }
}